fbw-a32nx/src/systems/fmgc/src/flightplanning/new/waypoints/WaypointFactory.ts

```diff
@@ -61,4 +61,12 @@ export namespace WaypointFactory {
         };
     }
 
+    export function fromAirport(airport: Airport): Waypoint {
+        return {
+            ...airport,
+            sectionCode: SectionCode.Enroute,
```

Comment:
Wrong section code for an airport. Should be section code = "P" (airport), subsection code = "A" (reference point).